self-api/src/main/java/com/selfxdsd/api/Admin.java

```diff
@@ -0,0 +1,16 @@
+package com.selfxdsd.api;
```

Comment:
@nmonokov Can you add a licence header to this file? Just copy it from other java files.